[2021-06-06 14:15:36]getData.py(304): Process Start
